File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    m = {
        'G': 'C',
        'C': 'G',
        'T': 'A',
        'A': 'U'
    }
    rna = ''
    for c in dna_strand:
        rna += m[c]
    return rna

File: users/aaronddvsa/javascript/rna-transcription/rna-transcription.js

export const toRna = (stringToTest) => {

  let input = stringToTest.split("");
  let rna = "";

  let nucleotides = {
    'C': 'G',
    'G': 'C',
    'T': 'A',
    'A': 'U',
  }

  for (let i = 0; i < input.length; i++) {
    rna += nucleotides[input[i]];
  }

  return rna;
};
